### PR TITLE
Downsample sample added. Import a ZDF point cloud and downsample it.

### PR DESCRIPTION
**Pull request by SatjaSivcev**

### CODE REVIEW

**Comment by SatjaSivcev** [comment]
Ready for the next round. Review the whole README file, please.

File: README.md [hunk 1]
```diff
@@ -41,7 +48,7 @@ git clone https://github.com/zivid/cpp-extra-samples
...
+Choose a sample solution, configure it with CMake, open it in Visual Studio, build it, run it. If you are uncertain about doing this, check out our tutorials for configuring [**Zivid**](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/61472793/Configure+Zivid+Extra+Samples+with+CMake+and+build+them+using+Visual+Studio+in+Windows), [**Zivid & PCL**](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/44204066/Configure+C+Zivid+Point+Cloud+Library+PCL+Samples+with+CMake+and+build+them+using+Visual+Studio+in+Windows), and [**Downsample**](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/76480513/Configure+C+Downsample+Sample+with+CMake+and+build+it+using+Visual+Studio+in+Windows) samples with CMake and building them using Visual Studio in Windows.
```
Comment (knatten) [inline README.md]: The Downsample link tells me the page is restricted, I assume you plan on publishing it after the PR is merged?

File: README.md [hunk 2]
```diff
@@ -23,6 +24,12 @@ Check out our [tutorial on configuring and building these samples on Ubuntu](htt
 	- [**ReadPCLVis3D**](https://github.com/zivid/cpp-extra-samples/blob/master/ZividPCL/ReadPCLVis3D/ReadPCLVis3D.cpp) - This example shows how to read a PCL point cloud and visualize it.
 	- [**CaptureWritePCLVis3D**](https://github.com/zivid/cpp-extra-samples/blob/master/ZividPCL/CaptureWritePCLVis3D/CaptureWritePCLVis3D.cpp) - This example shows how capture a Zivid point cloud, save it to a .PCD file format, and visualize it.
 	- [**CaptureFromFileWritePCLVis3D**](https://github.com/zivid/cpp-extra-samples/blob/master/ZividPCL/CaptureFromFileWritePCLVis3D/CaptureFromFileWritePCLVis3D.cpp) - This example shows how capture a Zivid point cloud from an emulated Zivid camera, save it to a .PCD file format, and visualize it.
+	- **Dependencies:**
+		- [PCL](http://pointclouds.org/) version 1.9.1
+
+- [**Downsample**](https://github.com/zivid/cpp-extra-samples/tree/master/Downsample)
```
Comment (knatten) [inline README.md]: The other samples have a short text explaining what they're about, maybe this one should have that as well?

File: README.md [hunk 3]
```diff
@@ -23,6 +24,12 @@ Check out our [tutorial on configuring and building these samples on Ubuntu](htt
...
+	- **Dependencies:**
+		- [Eigen](http://eigen.tuxfamily.org/) version 3.3.90
```
Comment (knatten) [inline README.md]: I guess we don't require exactly this version? Probably 3.4.0 will work also etc. Maybe just say "version 3.3.90 or newer" or something like that. Managing versions of dependencies properly is hard, but I think that should be good enough for a sample like this.

File: README.md (hunk 1 above)
Comment (SatjaSivcev) [reply README.md]: Yes, that is correct.

File: README.md (hunk 2 above)
Comment (SatjaSivcev) [reply README.md]: Description added.

File: README.md (hunk 3 above)
Comment (SatjaSivcev) [reply README.md]: Done

**Pull request merged**